### PR TITLE
event: Add support for offers and performer

### PR DESCRIPTION
**Pull request by Roohn**
## Description of Change(s):
Added support for offers and performer on the event jsonld (see: https://developers.google.com/search/docs/data-types/event#structured-data-type-definitions)
I moved the offer and aggregateOffer to a reusable file because it is also used on the product jsonld.

Please note that this is my first contribution, so any feedback is welcome!

---

Some things to help get a PR reviewed and merged faster:

- Have you updated the documentation to go with your changes? ✅ 
- Have you written or updated unit tests? ✅ 
- Have you written an integration test in the test app supplied? ✅ 

The above are generally required on all PR's.

Please have a read of the Contributing Guide for full details.

https://github.com/garmeeh/next-seo/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/utils/buildAggregateOffer.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { AggregateOffer } from '../types';
+
+export default (offer: AggregateOffer) => `
```
Comment (garmeeh) [inline src/utils/buildAggregateOffer.ts:3]: Could you change this to be a named export. 

```
export const  buildAggregateOffer(
```

File: src/utils/buildOffers.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Offers } from '../types';
+
+// TODO: Docs for offers itemCondition & availability
+// TODO: Seller type, make dynamic
+export default (offers: Offers) => `
```
Comment (garmeeh) [inline src/utils/buildOffers.ts:5]: Same here please.

```
export const buildOffers(
```

File: src/meta/__tests__/buildTags.spec.tsx [hunk 3]
```diff
@@ -303,7 +303,8 @@ it('displays defaultTitle when no title is provided', () => {
   const title = getByText(
     container,
     (content, element) =>
-      element.tagName.toLowerCase() === 'title' && content.startsWith(defaultTitle),
```
Comment (garmeeh) [inline src/meta/__tests__/buildTags.spec.tsx:306]: Looks like formatting change? Lets revert.

**Review by garmeeh — CHANGES_REQUESTED** [review]
Thanks for the contribution 😄 

Just a few minor suggested changes.

File: src/utils/buildAggregateOffer.ts (hunk 1 above)
Comment (Roohn) [reply src/utils/buildAggregateOffer.ts]: Ofcourse! I have updated it 👍🏼

**Comment by garmeeh** [comment]
Hey @Roohn, this looks good to me. Could you address the conflicts and I will get this approved and released.

**Comment by Roohn** [comment]
Hey @garmeeh I have rebased the branch and addressed all conflicts 👍🏼

**Comment by garmeeh** [comment]
@all-contributors please add @Roohn doc code test

**Review by garmeeh — APPROVED** [review]
😄 Thanks for the contribution.

**Pull request merged**